.circleci/config.yml

```diff
@@ -8,31 +8,37 @@ executors:
 
 # reusable command for all jobs
 commands:
-  checkout_from_cache:
-    description: 'To checkout and restore the dependencies cache'
+  checkout_from_workspace:
+    description: 'To checkout and attach the workspace'
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - dependency-cache-{{ checksum "package.json" }}
-            - dependency-cache-
+      - attach_workspace:
+          at: ~/webex-components
 
 # define jobs
 jobs:
   install:
     executor: main-executor
     steps:
-      - checkout_from_cache
+      - checkout
+      - restore_cache:
```

Comment:
Restoring before `npm ci` is meaningless, since `npm ci` wipes `node_modules` before installing.
I removed the restore before installing dependencies in my PR and just have it `save_cache` after

Reply:
then what's the point of having a cache. I think I figured what to do now.

Reply:
`save_cache` caches between triggers. So everytime circleci runs it will use the same node_modules cache. This "could" be used for the deploy job since there isn't an expectation of new modules being installed at that time and the checksum on `package-lock` should be the same

Reply:
sure, your example save the cache but you are not using/restoring the cache if you remove `restore_cache`. And that does NOT serve the purpose of caching. and thats why saving the cache and not restoring it is as good as not having the cache in the first place. Anyways, as I mentioned earlier I figured what to do. But I do suggest you read more about CircleCI configuration here: https://circleci.com/docs/2.0/configuration-reference/#save_cache
<img width="970" alt="Screen Shot 2019-08-31 at 11 22 23 AM" src="https://user-images.githubusercontent.com/14226615/64067698-c255e500-cbe1-11e9-9f44-c9415c741d9d.png">